### PR TITLE
Ncsdk 3898 links.txt

### PR DESCRIPTION
**Pull request by ru-fu**
Adding links.txt and shortcuts.txt.
This PR needs a PR in the nrf repo for integration - https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/2153

### CODE REVIEW

File: ble_controller/README.rst [hunk 1]
```diff
@@ -1,17 +1,17 @@
 .. _ble_controller:
 
 nRF Bluetooth LE Controller
-===========================
+###########################
 
-| The nRF Bluetooth LE Controller is an RTOS-agnostic library built for the Nordic Semiconductor nRF52 and nRF53 Series that supports Bluetooth 5.
-| In addition to providing a BLE HCI interface, it also provides:
+The |controller| is an RTOS-agnostic library built for the Nordic Semiconductor nRF52 and nRF53 Series that supports Bluetooth 5.
+In addition to providing a Bluetooth LE HCI interface, it also provides:
```
Comment (rugeGerritsen) [inline ble_controller/README.rst:7]: The Bluetooth terminologi is just `Bluetooth HCI interface`. I would therefore prefer to keep it as is.

Comment (ru-fu) [reply ble_controller/README.rst:7]: We should change to Bluetooth HCI interface then, not leave as BLE HCI interface?

File: doc/links.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+.. _`nRF9160 modem firmware zip file`: https://www.nordicsemi.com/Products/Low-power-cellular-IoT/nRF9160/Download#98C9E2578566420786ABD40B695FDB9B
+
+.. _`mbed TLS backend`: https://tls.mbed.org/
+
+.. _`article on reducing mbed TLS memory and storage footprint`: https://tls.mbed.org/kb/how-to/reduce-mbedtls-memory-and-storage-footprint
+
+
+
+.. ### Links to nrf (not a good way to link)
```
Comment (b-gent) [inline doc/links.txt:19]: We should discuss whether this is the preferred way of linking to nrf, or, for example, absolute links to 'latest'.

**Comment by b-gent** [comment]
Looks great, thanks!

File: doc/links.txt (hunk 2 above)
Comment (ru-fu) [reply doc/links.txt:19]: Those are then even more likely to break. :(
Ideally, we should set up intersphinx (see https://projecttools.nordicsemi.no/jira/browse/NCSDK-3934).

**Comment by rlubos** [comment]
Merging, according to @ru-fu the doc CI failure is expected in this PR.

**Pull request merged**